src/index.html

```diff
@@ -2,12 +2,178 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Uber Eats</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
-  </body>
+  <header>
+    <div class="logo-block" >
+      <a href="#"><img class="logo-img" src="./images/logo.svg" alt="logo_uber"></a>
+      <div class="delivery-block">
+        <lable>
+          <input class="speed-deliv"  type="text" placeholder="ASAP">
+        </lable>
+        <span class="to">to</span>
+        <label class="delivery-point">
+          <img src="./images/location.svg" alt="">
+          <input class="delyvery-type" type="text" placeholder="ul. Tarasivska St, 16">
```

Comment:
Не уверен, что type подходящее слово
```suggestion
          <input class="delivery-type" type="text" placeholder="ul. Tarasivska St, 16">
```